frontend/js/src/metadata-viewer/MetadataViewer.tsx

```diff
@@ -373,7 +394,11 @@ export default function MetadataViewer(props: MetadataViewerProps) {
               aria-labelledby="headingTwo"
             >
               <div className="panel-body">
-                <TagsComponent tags={metadata?.tag?.release_group} />
+                <TagsComponent
```

Comment:
The other TagComponents have a `{Boolean(metadata?...) &&` before them, one is needed here as well probably.

Reply:
Yep, I've got that currently in my local branch but not yet pushed, encountered some issues with stale data, still finishing that up.